### PR TITLE
Feat: suspendedChains added to bws config

### PR DESCRIPTION
**Pull request by Gamboster**

### CODE REVIEW

**Review by matiu — CHANGES_REQUESTED** [review]

File: packages/bitcore-wallet-service/src/config.ts [hunk 1]
```diff
@@ -82,6 +82,7 @@ module.exports = {
   maintenanceOpts: {
     maintenanceMode: false
   },
+  bchSuspended: false,
```
[the remark this answers is not in the document]
Comment (Gamboster) [reply packages/bitcore-wallet-service/src/config.ts]: much better 👍

File: packages/bitcore-wallet-service/src/lib/errors/errordefinitions.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { ClientError } from './clienterror';
 const errors = {
   AD_ALREADY_EXISTS: 'Ad already exists',
   BAD_SIGNATURES: 'Bad signatures',
+  BCH_SUSPENDED: 'BCH network is upgrading. Please try later.',
```
[the remark this answers is not in the document]
Comment (Gamboster) [reply packages/bitcore-wallet-service/src/lib/errors/errordefinitions.ts]: done

File: packages/bitcore-wallet-service/src/lib/server.ts [hunk 3]
```diff
@@ -2188,6 +2188,8 @@ export class WalletService {
 
           if (wallet.scanStatus == 'error') return cb(Errors.WALLET_NEED_SCAN);
 
+          if (wallet.coin == 'bch' && config.bchSuspended == true) return cb(Errors.BCH_SUSPENDED);
```
[the remark this answers is not in the document]
Comment (Gamboster) [reply packages/bitcore-wallet-service/src/lib/server.ts]: done

**Review by matiu — APPROVED** [review]
Great job!

**Pull request merged**